news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
@grangeway, may I also suppose the mentioned files are distributed under GPL as the part of  GPL-software, while inside none of them, or anywhere in the project, other copyright and/or license notice present?
As a fair contributor and/or user, should I investigate derivation of the code in the files, and if found to be proprietary, accept the corresponding non-free license and be somehow obliged to the owner?
It is very sad the things have turned this way.
I hope an unharmful solution will be found, after all the author decided to retire Ace Admin, he or she might release it under a GPL-compatible license.